.flake8

```diff
@@ -0,0 +1,32 @@
+[flake8]
```

Comment:
Needed to run locally and in VENV, added some exceptions for long lines, comments (to keep ansible boilerplate).